libs/application/templates/party-letter/src/fields/MinistryOfJusticeOverview/index.tsx

```diff
@@ -15,7 +15,7 @@ const mapToCSVFile = (endorsements: Endorsement[]) => {
   return endorsements.map((endorsement) => {
     return {
       Kennitala: formatKennitala(endorsement.endorser),
-      Dagssetning: format(new Date(endorsement.created), 'dd.MM.yyyy'),
+      Dagsetning: format(new Date(endorsement.created), 'dd.MM.yyyy'),
```

Comment:
lol thanks!
but since you're touching on the party-letter, would you add these strings to messages (kennitala, dagsetning etc) and add 2 columns, one for invalidated boolean check and one for bulk imported.. maybe call them "Í vafa" and "Meðmæli á pappír" and check them with and "x" if true and empty otherwise 👼